pkg/internal/checkup/checkup.go

```diff
@@ -23,6 +23,7 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"strings"
```

Comment:
Please consider moving this commit to the beginning, so the flow of serial console changes will not have a context switch.

Reply:
DONE